lumber-generate.js

```diff
@@ -10,6 +10,7 @@ const logger = require('./services/logger');
 program
   .description('Generate a backend application with an ORM/ODM configured.')
   .option('-c, --connection-url <connectionUrl>', 'Enter the database credentials with a connection URL')
+  .option('-H, --application-host <applicationHost>', 'Hostname of your admin backend app')
```

Comment:
[Design] According to your epic specification `-H` should be `-h`

Reply:
`-h` is already in use (see `--help`) in all lumber commands, thanks to `command`:

```
➜  ~ lumber generate -h
Usage: lumber-generate [options]

Generate a backend application with an ORM/ODM configured.

Options:
  -c, --connection-url <connectionUrl>  Enter the database credentials with a connection URL
  --no-db                               Use Lumber without a database.
  -h, --help                            output usage information
```

One character parameters are inconsistent by nature, I thought `H` was a good candidate. It is "host" first character. Feel free to suggest another shorthand.

Reply:
Ha Sorry, did think about this constraint. You're god damn right!